tests/proxy_test.go

```diff
@@ -140,7 +145,9 @@ func TestProxyHandleDialError_GRPC(t *testing.T) {
 	}
 }
 
-func TestProxyHandle_DoneContext_GRPC(t *testing.T) {
+func TestProxyHandle_ContextDeadlineExceeded_GRPC(t *testing.T) {
+	defer goleak.VerifyNone(t, goleak.IgnoreCurrent())
```

Comment:
nit: I'd add a comment saying we need to IgnoreCurrent due to klog leaks

Reply:
Good point, will add that